client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -61,9 +62,18 @@ const PatternAssembler: Step = ( { navigation, flow } ) => {
 	};
 
 	useEffect( () => {
-		// Require to start the flow from the first step
 		if ( ! selectedDesign ) {
-			goToStep?.( 'goals' );
+			setSelectedDesign( {
+				slug: 'blank-canvas-3',
+				title: 'Blank Canvas',
+				recipe: {
+					stylesheet: 'pub/blank-canvas-3',
+				},
+				verticalizable: false,
+				is_premium: false,
+				is_bundled_with_woo_commerce: false,
+				design_type: 'assembler',
+			} as Design );
```

Comment:
@Automattic/lego this might conflict with your work done in https://github.com/Automattic/wp-calypso/pull/71852. This was needed to allow users coming from the themes showcase to start the pattern assembler flow while skipping the rest of the site setup flow. Do you see any issues with this approach?

Reply:
Thanks for the ping, Miguel!

We added that redirection to the `goals` step to ensure users _creating a new site_ don't skip the goals step if they access the pattern assembler URL directly. Now that there are other use cases like yours to access the assembler step directly on existing sites, we'll have to add an exception to that check and create a new flow with only the assembler.

For now, rather than remove the redirection you could use a URL param to identify the new use case and skip the redirection. In the future, we could use the new assembler flow name to skip the redirection. I'll keep you updated.

Reply:
> URL param to identify the new use case

I think this would be a better idea for now

Reply:
This has been already addressed in https://github.com/Automattic/wp-calypso/pull/72747, so I'll revert these changes.